update docs for notification (#27733)
airbytehq/airbyte: docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md

This page provides guidance on how to manage notifications for Airbyte Cloud, allowing you to stay up-to-date on the activities in your workspace. 

## Configure Notification Settings

To set up Webhook notifications:

1. On the [Airbyte Cloud](http://cloud.airbyte.com) dashboard, click **Settings**.

2. Click **Notifications**.

> 3. Have a Slack Webhook ready if you plan to use Slack Webhook notification. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).

Reviewer: ```suggestion
3. Have a Slack Webhook ready if you plan to use Slack Webhook notifications. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
```

Reviewer 2: A webhook outside of Slack is also acceptable. Could we make this more generic?

Could rephrase to: 3. Have a webhook URL ready if you plan to use webhook notifications. Using a Slack webook is recommended. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).

Author: done!  one thing to note is the webhook notification we sent might not be consumed by other endpoints other than slack; slack requires a json format when sending webhook message, though it's not strange nor specific to slack;

Reviewer 2: Ah, okay good to know!

> 4. Toggle type of events you are interested to receive notifications. 

Reviewer: ```suggestion
4. Toggle the type of events you are interested to receive notifications for. 
```

> 	1. If using slack webhook, please fill in your webhook address. For your convenience, we provide a 'test' function to send a test message to your slack channel so you can make sure everything has been connected.

Reviewer: ```suggestion
	1. If using slack webhooks, the webhook URL is required. For your convenience, we provide a 'test' function to send a test message to your slack channel so you can make sure it's working as expected.
```

> 	1. If using slack webhooks, the webhook URL is required. For your convenience, we provide a 'test' function to send a test message to your slack channel so you can make sure it's working as expected.

Reviewer 2: Wording suggestion: To enable webhook notifications, the webhook URL is required. For your convenience, we provide a 'test' function to send a test message to your webhook URL so you can make sure it's working as expected.

Could you also add another sub-bullet that says: "Enabled email notifications will send an email to all users of the workspace."

Reviewer 2: On the note about email notifications - that's my understanding of how this will work but please correct me if I'm wrong!

Author: Right - no, email notifications at this moment will only be sent to the owner of the workspace.

Reviewer 2: Oh interesting. Okay thanks.

5. Click **Save changes**.

## Notification Event Types

> 1. Failed syncs and Successful syncs: When a connection sync has finished, you can choose to be notified with if the sync has failed, successful or both. Note if sync runs frequently or if there are many syncs in the workspace these types of events can be noisy.

Reviewer: ```suggestion
1. Failed syncs and successful syncs: When a connection sync has finished, you can choose to be notified whether the sync has failed, succeeded or both. Note that if sync runs frequently or if there are many syncs in the workspace these types of events can be noisy.
```

> 1. Automated Connection Updates: when a connection detects source schema has changed and can be updated automatically, Airbyte will update you connection and send you a notification message.

Reviewer: ```suggestion
1. Automated Connection Updates: when Airbyte detects that the connection's source schema has changed and can be updated automatically, Airbyte will update your connection and send you a notification message.
```

> 1. Automated Connection Updates: when Airbyte detects that the connection's source schema has changed and can be updated automatically, Airbyte will update your connection and send you a notification message.

Reviewer 2: The numbers here are all 1. Could we get those numbered in sequence?

Author: that's markdown format - after rendering it should look as expected - you can verify it in https://markdownlivepreview.com/

Reviewer 2: Oh okay, makes sense!

> 1. Connection Updates Requiring Action: When a connection detects some updates that requires action to run syncs. Since this will affect your sync from running as scheduled, you cannot disable the email notification for this type of event.

Reviewer: ```suggestion
1. Connection Updates Requiring Action: When Airbyte detects some updates that requires your action to run syncs. Since this will affect your sync from running as scheduled, you cannot disable this type of email notification.
```

> 1. Sync Disabled and Sync disabled Warning: If a sync has been failing for multiple days or many times consecutively, we will disable the connection to prevent it run further. We will send a sync disabled warning notification when we detects the trend, and once the failure counts hits our threshold we will send a sync disabled notification and will actually disable the connection. Again, because the sync will not continue to run as you have configured, we made the decision that you cannot disable the email notification for 'Sync Disabled' event so you will always get notified.

Reviewer: ```suggestion
1. Sync Disabled and Sync Disabled Warning: If a sync has been failing for multiple days or many times consecutively, Airbyte will disable the connection to prevent it from running further. Airbyte will send a Sync Disabled Warning notification when we detect the trend, and once the failure counts hits a threshold Airbyte will send a Sync Disabled notification and will actually disable the connection. Again, because the sync will not continue to run as you have configured, the Sync Disabled notification cannot be disabled.
```

 

## Enable schema update notifications

To get notified when your source schema changes: 

> 1. Make sure you have `Automatic Connection Updates` and `Connection Updates Requiring Action` turned up for desired notification channels; If these are off, even if you turned up the settings on connection settings, Airbyte Cloud will *NOT* send out any notifications related to these types of events.

Reviewer: I'm assuming this is not cloud-specific, but I could be wrong:

```suggestion
1. Make sure you have `Automatic Connection Updates` and `Connection Updates Requiring Action` turned on for your desired notification channels; If these are off, even if you turned on schema update notifications in a connection's settings, Airbyte will *NOT* send out any notifications related to these types of events.
```
